src/styles/common/_animations.scss

```diff
@@ -7,21 +9,3 @@
         opacity: 1;
     }
 }
-
-@mixin bdl-fadeIn($fade-in-time) {
```

Comment:
Is this used anywhere else or in any other projects? If so, removing it would be a breaking change.

Reply:
i searched and only saw internal references to it here, and no references in known projects. I think it should technically be a breaking change, but wanted to discuss. Two options stand out:
 
* mark this as breaking with a note on why
* `@include` the renamed and moved mixin here with a [`warn`](https://sass-lang.com/documentation/at-rules/warn) saying what to use instead.